Directory.Packages.props

```diff
@@ -0,0 +1,22 @@
+﻿<Project>
+	<ItemGroup>
+    	<PackageVersion  Include="BenchmarkDotNet" Version="0.13.1" />
+		<PackageVersion  Include="D2L.CodeStyle.Analyzers" Version="0.142.0" />
+		<PackageVersion  Include="D2L.CodeStyle.Annotations" Version="0.28.0" />
+		<PackageVersion  Include="D2L.Services.Core.Exceptions" Version="2.0.1.15" />
+		<PackageVersion  Include="FluentAssertions" Version="6.2.0" />
+		<PackageVersion  Include="Microsoft.AspNet.WebApi.Core" Version="[5.2.7]" />
+		<PackageVersion  Include="Microsoft.AspNet.WebApi.OwinSelfHost" Version="[5.2.7]" />
```

Comment:
How come these two are pinned? I doubt there will be any future releases since it was last updated in 2018, but still feels a little strange.

Reply:
Uh good question, those are actually the latest.

I got an unrelated build error and added this but it presumably did nothing. I was fine doing that because yikes @ those packages in general (the D2L.Security.OAuth2.WebApi package is effectively "deprecated" at this point, but there is still one user...)